### PR TITLE
feat(html pipe): Unwrap hero images

### LINKED ISSUE #338: <img> gets wrapped by a <p> tag
When adding an image to markdown, it gets generated and wrapped by a `<p>` tag. This is not semantic and it's invalid HTML. (It's valid to put a block element inside of another block element. But rendering an empty `<p>` tag with an `` tag inside is not semantic

Suggestion: `` tags generated should not be wrapped by `<p>`

### PR DESCRIPTION
**Pull request by ramboz**
Unwrap sole images in a single paragraph in a section so the hero image is not inside a paragraph
tag

fix #338

### CODE REVIEW

**Comment by ramboz** [comment]
With this fix, markdown like the following:
```markdown
![Foo](foo.jpeg)
```
or:
```markdown
# Bar

Baz
![Qux](qux.jpeg)

---

![Foo](foo.jpeg)

---

## Corge

Grault
```
will be respectively converted to:
```html
<img src="foo.jpeg" alt="Foo"/>
```
and:
```html
<h1>Bar</h1>
<p>Baz
  <img src="qux.jpeg" alt="Qux"/>
</p>
<hr> <!-- // will be a proper section with #239 -->
<img src="foo.jpeg" alt="Foo"/>
<hr> <!-- // will be a proper section with #239 -->
<h2>Corge</h2>
<p>Grault</p>
```

File: src/html/unwrap-hero-images.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
```
Comment (tripodsan) [inline src/html/unwrap-hero-images.js]: 2019...

File: src/html/unwrap-hero-images.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const map = require('unist-util-map');
+
+function reformat({ content }) {
```
Comment (tripodsan) [inline src/html/unwrap-hero-images.js]: maybe add an explanation here....

**Comment by ramboz** [comment]
@trieloff Done!

**Pull request merged**